### PR TITLE
minor fixes for things I noticed when using AWS

### PR DESCRIPTION
**Pull request by ecpierce**

### CODE REVIEW

File: README.md
Comment (taylorreiter) [inline README.md]: so you need to do this regardless, or to be able to view the streamlit launch in browser?

File: bsoid_app/extract_features.py
Comment (taylorreiter) [inline bsoid_app/extract_features.py]: why comment this out? should it be fully removed?

**Review by taylorreiter — COMMENTED** [review]
unless you urgently need an approval, going to just comment and perhaps wait for Mert on these things -- I think this stuff could impact downstream deployment even tho the actual amount of changes is very small

File: README.md
Comment (taylorreiter) [reply README.md]: Can you add that this is an inbound rule and that the python script runs the streamlit app? Also maybe add the context that the user will be able to use the external URL that b-soid prints to have a log of what's running -- if b-soid fails, the error messages appear there, not inline in the terminal, sometimes.

File: run_streamlit_cli.py
Comment (mertcelebi) [inline run_streamlit_cli.py]: I'd maybe only include this in the README. And link to the README from here, so you don't have duplicate it and change this in multiple places

File: README.md
Comment (mertcelebi) [reply README.md]: What Taylor said. I'd note 1) if you are running this on AWS AND need to see the web application (which is useful for debugging since errors sometimes only appear in the web-app), you need to set inbound rule. 2) That the inbound rule port is 8501 but that the users should be careful to make sure that this is the right port when they run the app (for example, if you incorrectly shutdown so the process is running the background and you run the streamlit app again, it'll select the next port).

Comment (mertcelebi) [reply README.md]: This should not be an issue in VSCode technically but a useful note regardless.

File: README.md
Comment (mertcelebi) [inline README.md]: I'd note how it speeds up runtime. It's not very clear what non-repeated examples means.

File: bsoid_app.py
Comment (mertcelebi) [inline bsoid_app.py:72]: image

File: bsoid_app/extract_features.py
Comment (mertcelebi) [inline bsoid_app/extract_features.py]: I'd not comment these out since they change the streamlit behavior and we actually do use streamlit a fair bit (for debugging/training purposes)

**Review by mertcelebi — CHANGES_REQUESTED** [review]
A couple suggestions

File: bsoid_app.py
Comment (ecpierce) [reply bsoid_app.py:72]: sorryyyy

File: bsoid_app/extract_features.py
Comment (ecpierce) [reply bsoid_app/extract_features.py]: put back

File: bsoid_app/machine_learner.py
Comment (mertcelebi) [inline bsoid_app/machine_learner.py]: I would keep these uncommented actually!

**Review by mertcelebi — APPROVED** [review]
Approving with a smol comment

File: bsoid_app/machine_learner.py
Comment (ecpierce) [reply bsoid_app/machine_learner.py]: oopsies! fixed

**Pull request merged**
